src/components/filterBox/FilterBox.tsx

```diff
@@ -43,6 +44,10 @@ export class FilterBox extends React.Component<IFilterBoxProps, any> {
     this.filterInput.value = nextInputValue;
     this.filterInput.nextElementSibling.setAttribute('class', this.filterInput.value.length ? '' : 'hidden');
 
+    if (this.props.formatFilter) {
+      this.props.formatFilter(this.props.id, this.filterInput.value);
```

Comment:
you could just name this `onFilterCallback` since it could be use for other purposes...just like you did in other components...just a more generic name